dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```diff
@@ -69,7 +69,12 @@ RUN apt-get update -y && \
         libcairo2\
         libasound2\
         libatspi2.0-0\
-        libxshmfence1 && \
+        libxshmfence1 \
+        fonts-arphic-ukai \
```

Comment:
I am curios about the list we arrived at here, We don't even have the basic one `fonts-liberation` for the image.

Do we know if that works? 

Here is the list from PW  for comparison - https://github.com/microsoft/playwright/blob/5ba7903ba098586a13745e0d7ac894f1d55d47aa/packages/playwright-core/src/utils/nativeDeps.ts#L434-L470

One we use in RUM agent for puppeteer and playwright - https://github.com/elastic/apm-agent-rum-js/tree/master/.ci/docker

Reply:
Here I followed an approach of testing websites and taking note of which didn't display all characters correctly. Then, I went after the packages only for those, and ran an E2E test which generated the images above. I followed this process to avoid adding too many unnecessary packages.

Wrt to fonts-liberation, it seemed to be a standard set of fonts which could just replace the default set of fonts to which it's compatible, but we already have default fonts for western characters in the base image and thus I thought we wouldn't need that. Looking into the other packages, they also seem to fall into one of two categories:
* Covering standard character sets (like `fonts-liberation` or `ttf-unifont`)
* Tools for managing fonts themselves (like `libfontconfig`)

The other fonts seemed to be the most complete _and_ popular fonts for their character standards (`fonts-arphic-ukai` [for chinese characters] , `fonts-ipafont-mincho` [for japanese characters, `fonts-ipafont-gothic` being for sans-serif japanese characters], and `fonts-unfonts-core` for Korean characters). Those were the ones which came up when I looked for packages for each individual language.

We already had support for Arabic/Russian characters in the base image as per the initial test, so I haven't added those as the websites I visited already displayed correctly. It could be the case that there are characters outside the unicode space 

Looking at Playwright's deps these are the changes I see we could make:
* [`fonts-wqy-zenhei`](https://packages.debian.org/sid/fonts-wqy-zenhei) this seems like a more complete font set to replace the one in the package we have. However, I was just thinking about the licensing of these given that [this package seems to be GPL licensed](http://wenq.org/wqy2/index.cgi?ZenHei(en)) and therefore I don't know whether we could include it in the image we distribute. The [Arphic Public License](https://www.freedesktop.org/wiki/Arphic_Public_License/) for the current fonts seem to only apply for modification of the fonts themselves and is incompatible with GPL and thus seems fine, but actually it would be good to review.
    ➡️  @andrewvc @vigneshshanmugam do we have a particular specialist in software licensing to review these?
* [`fonts-tlwg`](https://linux.thai.net/projects/fonts-tlwg) seems to support Thai characters, so that's one I think we could add. [However, it also includes GPL licensed fonts, so we must double check that](https://github.com/tlwg/fonts-tlwg/blob/master/COPYING).
* [`fonts-noto-color-emoji`] for supporting Google's emoji set (which should really follow the same set as the standard, shouldn't it?)

I moved this PR to `draft` just so that we would make sure to review the licensing and whether we're happy with the current coverage set. I do thing we should add `fonts-tlwg` though, so I'll run a test for that before opening this PR for review again too.